salt/utils/aws.py

```diff
@@ -61,6 +63,20 @@
 __AssumeCache__ = {}
 
 
+def sleep_exponential_backoff(attempts):
+    """
+    backoff an exponential amount of time to throttle requests
+    during "API Rate Exceeded" failures as suggested by the AWS documentation here:
+    https://docs.aws.amazon.com/AWSEC2/latest/APIReference/query-api-troubleshooting.html
+    and also here:
+    https://docs.aws.amazon.com/general/latest/gr/api-retries.html
+    Failure to implement this approach results in a failure rate of >30% when using salt-cloud with
+    "--parallel" when creating 50 or more instances with a fixed delay of 2 seconds.
+    A failure rate of >10% is observed when using the salt-api with an asyncronous client
+    specified (runner_async)."""
```

Comment:
Can you move these closing quotes to it's own line? It's a little tricky to see where those end. Would you also mind changing those to single quotes instead of double? That's what we use in Salt's [style guide](https://docs.saltstack.com/en/latest/topics/development/conventions/style.html).